File: 1832-check-if-the-sentence-is-pangram/1832-check-if-the-sentence-is-pangram.go
func checkIfPangram(sentence string) bool {
    chars := make(map[rune]struct{})
    
    for _, c := range strings.ToLower(sentence) {
        chars[c] = struct{}{}
    }
    
    for _, c := range "abcdefghijklmnopqrstuvwxyz" {
        if _, ok := chars[c]; !ok {
            return false
        }
    }
    
    return true
}

File: 1832-check-if-the-sentence-is-pangram/1832-check-if-the-sentence-is-pangram.js
/**
 * @param {string} sentence
 * @return {boolean}
 */
var checkIfPangram = function(sentence) {
    const chars = {};
    
    for (const c of sentence.toUpperCase()) {
        chars[c] = true;
    }
    
    return [...'ABCDEFGHIJKLMNOPQRSTUVWXYZ'].every((c) => chars[c])
};